sign_language_segmentation/bin.py

```diff
@@ -59,15 +61,36 @@ def predict(model, pose: Pose):
 
 def get_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--pose', required=True, type=str, help='path to input pose file')
+    parser.add_argument('--pose', required=True, type=Path, help='path to input pose file')
     parser.add_argument('--elan', required=True, type=str, help='path to output elan file')
+    parser.add_argument('--save-segments', 
+                        type=str, 
+                        choices=["SENTENCE","SIGN"],
+                        help='whether to save cropped .pose files')
     parser.add_argument('--video', default=None, required=False, type=str, help='path to video file')
     parser.add_argument('--subtitles', default=None, required=False, type=str, help='path to subtitle file')
     parser.add_argument('--model', default='model_E1s-1.pth', required=False, type=str, help='path to model file')
     parser.add_argument('--no-pose-link', action='store_true', help='whether to link the pose file')
 
     return parser.parse_args()
 
+def save_pose_segments(tiers, tier_id, input_file_path):
```

Comment:
i would rather have get_args directly above main.
otherwise there is not reasonable continuity for the file